src/actor/character/styles/character/_main-tab.scss

```diff
@@ -41,8 +41,13 @@
   'initiative-modifier pronouns'
   'defense magic-defense'
   'hit-points mind-points'
+  'affinities .'
   'attributes-and-status-effects attributes-and-status-effects'
-  'traits traits';
+  'traits traits'
+  'attacks .'
+  'spells .'
+  'other-actions .'
+  'special-rules .';
```

Comment:
How similarly or differently are we going to handle monster attacks/spells/etc. from PCs? Do all these things apply to PCs?

Reply:
Yah we gonna need to have a product discussion on that one.  While not specifically in the book for PCs, I could see it being useful to have textareas like these for random other stuff we wanna keep track of.  I think PC attacks would be covered in the equipment tab?  And I also think PCs should have their own tab for spells.  You thinkin I should take out the areas not explicitly needed for character, and revamp them in the `monster/_main-tab.scss`?